### PR TITLE
Update README to reflect S3 policies

### PR DESCRIPTION
**Pull request by nlindblad**
We've made it possible to grant other team accounts access to the S3 objects mentioned in the README.

This update provides more information about that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,9 +73,12 @@ You should now be able to start the application (`sbt run`), go to [https://prof
 Install the local configuration file from s3:
 
 ```
+mkdir -p /etc/gu
 aws s3 cp --profile identity s3://gu-identity-frontend-private/DEV/identity-frontend.conf /etc/gu
 ```
 
+**Note**: If you do not have Janus access to Identity, we can grant your team specific access, which means you would substitute `--profile identity` with e.g. `--profile membership`.
```
Comment (markjamesbutler) [inline README.md]: "Contact the Identity team if you require access to these files."

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Web frontend for Sign in and Registration at [theguardian.com](http://theguardia
 Configuration files:
 - Environment-specific configuration (`conf/<ENV>.conf`)
 - Application configuration (`conf/application.conf`)
-- System file with additional properties (`/etc/gu/identity-frontend.conf`)
+- System file with additional properties (`/etc/gu/identity-frontend.conf` or `~/.gu/identity-frontend.conf`)
```
Comment (jamespamplin) [inline README.md]: I think we only read from `/etc/gu/identity-frontend.conf`, see: https://github.com/guardian/identity-frontend/blob/master/conf/DEV.conf#L18

But would be good to read from `~/.gu/` (in fact I actually map my `~/.gu/` to `/etc/gu`, so would be nice not to have to do this)

Comment (nlindblad) [inline README.md]: Okey. I'll look into reading from `~/.gu` as well. Cheers.

Comment (nlindblad) [inline README.md]: Added `~/.gu`

**Comment by jamespamplin** [comment]
:+1: looks good

**Pull request merged**